github/actions_workflow_jobs.go

```diff
@@ -0,0 +1,137 @@
+// Copyright 2020 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package github
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"net/url"
+)
+
+// Step represents a single task from a sequence of tasks of a job.
+type Step struct {
```

Comment:
In light of #1429, #79, #45, and #19, I'm going to make the executive decision and say that these two structs (`Job` and `Step`) should all use pointers.

In addition, please rename `Step` to be `TaskStep` to make it more clear when seeing it by itself.

And let's please change `Job` to `WorkflowJob` to make it more descriptive as well.

Thank you!

Reply:
Friendly ping, @joshuabezaleel to switch these fields to pointers, please.